src/net_processing.cpp

```diff
@@ -3273,8 +3273,9 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
             // - protocol version per the VERSION message supports WTXID_RELAY;
             // - we intended to exchange transactions over this connection while establishing it
             //   and the peer indicated support for transaction relay in the VERSION message;
-            // - we are not in -blocksonly mode.
-            if (pfrom.m_relays_txs && !m_ignore_incoming_txs) {
+            // - we are not in -blocksonly mode;
+            // - this is not a feeler.
+            if (pfrom.m_relays_txs && !m_ignore_incoming_txs && !pfrom.IsFeelerConn()) {
```

Comment:
What about `IsAddrFetchConn()` ? I think we should restrain ourselves to send `ConnectionType::ADDR_FETCH`.

Reply:
Similarly to #26396, i think this should be fixed above (`m_relays_tx`), and done in a separate PR :)
Let's hope Marco updates that one for now.